src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue

```diff
@@ -47,16 +47,16 @@
 <script>
 import FinneyModal from '../FinneyModal';
 import CustomerSupport from '@/components/CustomerSupport';
-import ledger from '@/assets/images/icons/button-ledger.png';
-import ledgerHov from '@/assets/images/icons/button-ledger-hover.png';
-import bitbox from '@/assets/images/icons/button-bitbox.png';
-import bitboxHov from '@/assets/images/icons/button-bitbox-hover.png';
-import secalot from '@/assets/images/icons/button-secalot.png';
-import secalotHov from '@/assets/images/icons/button-secalot-hover.png';
-import trezor from '@/assets/images/icons/button-trezor.png';
-import trezorHov from '@/assets/images/icons/button-trezor-hover.png';
-import keepkey from '@/assets/images/icons/button-keepkey.png';
-import keepkeyHov from '@/assets/images/icons/button-keepkey-hover.png';
+import ledger from '@/assets/images/icons/HardwareWallet/ledger.svg';
```

Comment:
you can probably remove the ones with "Hov" if you're importing the same images over..

Reply:
Removed